app/mailers/notifier_mailer.rb

```diff
@@ -81,10 +81,18 @@ def notify_new_finding_answer(users, finding_answer)
     mail to: users_to_notify_for(users).map(&:email),
          subject: prefix.upcase + t(
            'notifier.notify_new_finding_answer.title',
-           review: finding_answer.finding.review.to_s
+           review:     finding_answer.finding.review.to_s,
```

Comment:
```suggestion
           review:     finding_answer.finding.review,
```
Una tontera... jeje, pero te lo podés _ahorrar_, porque se llama automático al `to_s` en este caso (al menos en teoría).

Reply:
impecable franco, jajajaja a veces me pierdo con estas.